src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Teams/inviteModal.html.twig

```diff
@@ -7,16 +7,13 @@
                     <label for="email">{{ "teams.management.invite.modal.email.label"|trans }}</label>
                     <input type="text" name="email" id="email" />
                 </div>
-                <div class="form__row">
+                <div class="roles">
+                    <input type="hidden" name="role" value="member">
```

Comment:
Is it okay to set a default value here? This is something to discuss with the PM's. The entire modal is very UX unfriendly. There is no form validation and there is no flash message after the modal window performed his action.

Reply:
I feel the same way. But maybe we should run this by the PO? Either way I'm fine with this, we can fix it later if PO wants different behavior.